Deleting-A-Branch.md

```diff
@@ -0,0 +1,5 @@
+#Delete a Git branch both locally and remotely:
+
+```
+git push origin --delete <branchName>
+```
```

Comment:
[This](http://makandracards.com/makandra/621-git-delete-a-branch-local-or-remote) this has the right information needed.